src/resources/postcss/events-admin.pcss

```diff
@@ -292,12 +292,25 @@
 }
 
 .linked-post-wrapper {
+	tbody + tbody tr:first-child td {
+		padding-top: calc(var(--tec-spacer-4) + 2px);
```

Comment:
Typically, when faced with this I'd ask the designer to pick 20px or 24px 😉

Reply:
I'll pick `20px`!